projects/plugins/social/src/js/store/reducer/index.js

```diff
@@ -9,6 +9,7 @@ const reducer = combineReducers( {
 	siteData,
 	connectionData,
 	jetpackSettings,
+	showNudge: ( state = true ) => state,
```

Comment:
Rather than defining it separately and explicitly like this, should we just move the `showNudge` param to the `jetpackSettings`? Similar to `showPricingPage` and `hasPaidPlan`.

Reply:
Yes, I was thinking of doing that, and then thought it was too big a change for this stage, but actually it's probably not. `hasPaidPlan` is already a `jetpackSettings` selector. 

We should probably reverse the logic at some point anyway.